Detectors/MUON/MCH/Align/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+# All rights not expressly granted are reserved.
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+add_subdirectory(Aligner)
+if (BUILD_TESTING)
+add_subdirectory(Test)
```

Comment:
Hi Philippe,
I will delete the sub-directories, that directory was meant to be an area to put some macros for tests. 
About the workflow, you are totally right, for now it's not a "real workflow" since we are still trying to figure out if we finally put the whole alignment procedure inside a reconstruction chain and more precisely how to do it; so up to now we made a first version more like an executable for alignment uses. 
Chi.